### PR TITLE
doc(arrays): Arrays ds session notes added.

### LINKED ISSUE #18: Arrays
Arrays Data structure

### PR DESCRIPTION
**Pull request by shangan23**
#18 

Array data structure session notes.

### CODE REVIEW

File: Arrays.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+| Operation | Complexity | Comments|
+| ------------- | ------------- |------------- |
+| Get       | O(1) | Index is known to memory. No iteration required|
+| Set       | O(1) | Index is known to memory. No iteration required|
+| Insert       | O(n) | Index is known to memory. No iteration required|
+| - Begining       | O(n) | Iteration required to right shift the elements in an array|
+| - Middle       | O(n) | Iteration required to right shift the elements in an array|
+| - End       | O(1) | Index is known to memory. No iteration required (check reference:Amortized complexity) |
+| Delete       | O(n) | Index is known to memory. No iteration required|
+| - Begining       | O(n) | Iteration required to shift the elements in an array|
```
Comment (sriramanvellingiri) [inline Arrays.md]: @shangan23  Shift in the sense, it is left shift right  ?

File: Arrays.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    - End
+- Delete
+    - Beginning
+    - Middle
+    - End
+- Copy
+
+## What Static Array?
+
+![Static Arrays](images/ArrayCapacity.png "Static Arrays")
```
Comment (sriramanvellingiri) [inline Arrays.md]: @shangan23  Also create the separate image folder for array inside the image directory

**Review by sriramanvellingiri — CHANGES_REQUESTED** [review]
Change this requested

File: Arrays.md (hunk 1 above)
Comment (shangan23) [reply Arrays.md]: resolved

File: Arrays.md (hunk 2 above)
Comment (shangan23) [reply Arrays.md]: added folder

**Pull request merged**